cmd/utils/flags.go

```diff
@@ -858,6 +859,28 @@ var (
 		Name:  "rpc.getlogs.maxrange",
 		Usage: "Limit max fetched block range for `eth_getLogs` method",
 	}
+
+	// Da syncing settings
+	DASyncEnabledFlag = cli.BoolFlag{
+		Name:  "da.sync",
+		Usage: "Enable node syncing from DA",
+	}
+	defaultDa  = ethconfig.Defaults.DA.FetcherMode
```

Comment:
```suggestion
	defaultDA  = ethconfig.Defaults.DA.FetcherMode
```

it should always be `DA` or `da`, see here for more details on naming conventions for acronyms: https://google.github.io/styleguide/go/decisions#initialisms